tests/page_tests.py

```diff
@@ -236,14 +243,17 @@ def get_url_basename(url):
             else:
                 return url
 
-        def check_link(lnk, sess, filenames, folders):
-
-            if not check_skip_conditions(lnk, filenames, folders):
-                failure = try_to_connect(lnk, sess)
+        def check_link(link, session, filenames, folders):
+            if not check_skip_conditions(link, filenames, folders):
+                if get_url_basename(link) == "github.com" and any([f"{wiki.name}/wiki" in link for wiki in WIKI_INCLUDELIST]):
```

Comment:
The Wiki Checker currently runs with Python 2.7, which does not support format strings. Please use a different way to format strings.